README.md

```diff
@@ -1 +1,81 @@
-# ballerina-observability
\ No newline at end of file
+# Ballerina Observability
+
+Observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs. Monitoring, logging, and distributed tracing are key methods that reveal the internal state of the system to provide the observability. Ballerina becomes fully observable by exposing itself via these three methods to various external systems allowing to monitor metrics such as request count and response time statistics, analyze logs, and perform distributed tracing.
+
+## About Ballerina
+
+Ballerina makes it easy to write microservices that integrate APIs.
+
+#### Integration Syntax
+A compiled, transactional, statically and strongly typed programming language with textual and graphical syntaxes. Ballerina incorporates fundamental concepts of distributed system integration and offers a type safe, concurrent environment to implement microservices.
+
+#### Networked Type System
+A type system that embraces network payload variability with primitive, object, union, and tuple types.
+
+#### Concurrency
+An execution model composed of lightweight parallel worker units that are non-blocking where no function can lock an executing thread manifesting sequence concurrency. 
+ 
+## Table of contents
+
+- [Getting started](#getting-started)
+- [Download and install](#download-and-install)
+- [Contributing to Ballerina](#contributing-to-ballerina)
+- [License](#license)
+- [Useful links](#useful-links)
```

Comment:
Shall we add a section to list down the extensions available in this repository? And maybe link it to the ReadMe files inside each extension

Reply:
+1. Can you provide such a list?

Reply:
Added these in.